app/services/doc_auth/acuant/responses/get_results_response.rb

```diff
@@ -53,12 +53,17 @@ def response_info
         def create_response_info
           alerts = processed_alerts
 
+          failed_alert_results = alerts[:failed].each_with_object({}) do |alert, results|
+            results[alert[:name].downcase.parameterize(separator: '_').to_sym] = alert[:result]
+          end
```

Comment:
A few ideas
1. can we have a combined `alert_results` that includes logging both passed and failed alerts? It helps us look at the % of time each signal passes
2. should we be including the `side` for each of these so that the front vs back barcodes don't overwrite each other?
3. since I know that `processed_alerts` already has a layer of processing, what if we moved building this new combined thing directly in the `GetResultsResponse` class?

Reply:
So that it does not write over eachother side would have to be a key to the hash. 
for example if this is the hash         log_alert_results: { visible_pattern: 'Failed' },
Would it be log_alert_results: { back: { visiable_pattern: 'failed'}, front: { some_something: 'failed' } } 
but then what if there is no side?

Reply:
If there is no side, we wouldn't need to include it in the key, but many of these alerts do have sides, correct?

Reply:
would it cause an issue that the data structure sometimes includes a key and sometimes does not? If we do log_alert_results.keys  we get back, front but if no sides we get none. The tests dont all have sides. So I was going by that. I do not know if they usually have no side or not. Maybe we can put a key like "no_side" for the ones that do not have a side. That way the data structure is always the same. How do you feel about that? 
 { no_side: { visiable_pattern: 'failed'}, front: { some_something: 'failed' } }

Reply:
yes having `no_side` makes sense. but maybe we need to get updated fixtures to see if we can get a better sense of alerts